### PR TITLE
Doc IntelliJ OOME importing

### LINKED ISSUE #3438: Doc How to fix the OOM Exception when importing Quarkus into IntelliJ
This issue is the result  of the following discussion:
https://quarkusio.zulipchat.com/#narrow/stream/187038-dev/topic/IntelliJ.20OOME/near/171923769

### PR DESCRIPTION
**Pull request by danielpetisme**
Fix #3438 
It start also the FAQ section of the Contritbuting Doc.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -118,3 +118,16 @@ not being invalidated correctly in some cases. To run a build with a new instanc
...
+
+## Frequently Asked Questions
+
+* IntelliJ fails to import Quarkus Maven project with `java.lang.OutOfMemoryError: GC overhead limit exceeded` 
+
+In IntelliJ IDEA (version older than `2019.2`) if you see problems in the Maven view claiming `java.lang.OutOfMemoryError: GC overhead limit exceeded` that means the project import failed.
+  
+To fix the issue, you need to update the Maven importing settings:  
+`Build, Execution, Deployment` > `Build Tools`> `Maven` > `Importing` > `VM options for importer`
+To import properly Quarkus you need to set the Max Heap Size to 1Go: `-Xmx1g`
```
Comment (machi1990) [inline CONTRIBUTING.md]: It really depends, sometimes it can be up to 3Gs

Comment (geoand) [reply CONTRIBUTING.md]: Yeah, for me it's `768M` but I have heard of people talking about 2G or more. Perhaps you can make a note of it?

**Comment by geoand** [comment]
Can you please squash to a single commit?

**Review by geoand — APPROVED** [review]
Thanks for this!

**Pull request merged**